releasenotes/notes/apm-quantize-ip-addresses-on-peer-tags-f409b4e789a47875.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    For the [Inferred Service Dependencies beta](https://docs.datadoghq.com/tracing/guide/inferred-service-opt-in/?tab=java), ensure the same IP address quantization is done at the agent which is already being done on the backend during ingestion. This improves the performance of aggregation of stats on peer tags by reducing unnecessary aggregation which is currently being done on raw IP addresses. 
```

Comment:
```suggestion
    IP address quantization is done in the backend during ingestion. For the [Inferred Service Dependencies beta](https://docs.datadoghq.com/tracing/guide/inferred-service-opt-in/?tab=java), the Agent applies the same IP address quantization. This reduces unnecessary aggregation that is currently done on raw IP addresses. And therefore, improves the aggregation performance of stats on peer tags. 
```

Reply:
@maycmlee i accepted and modified slightly. Could you please review again?